vendor/bat-native-ads/src/bat/ads/internal/tokens/redeem_unblinded_token/create_confirmation_url_request_builder.cc

```diff
@@ -17,8 +17,6 @@
 #include "bat/ads/internal/server/confirmations_server_util.h"
 #include "bat/ads/internal/server/via_header_util.h"
 #include "bat/ads/internal/tokens/redeem_unblinded_token/create_confirmation_util.h"
-#include "brave/components/l10n/browser/locale_helper.h"
```

Comment:
dep can be removed?

Reply:
`#include "brave/components/l10n/browser/locale_helper.h"` is included in other files